index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>TripHouse</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <h1 class="header">What do me offer</h1>
+      <img class="line" src="./img/highlight (Stroke).jpg" alt="line" />
+      <div class="container">
+        <div class="content">
+          <img class="vektor" src="./img/Vector.png" alt="support" />
+          <p class="text">Support 24/7</p>
+        </div>
+        <div class="content">
+          <img class="vektor" src="./img/Vector (1).png" alt="communicate" />
```

Comment:
Vector (1) - realy bad practice, you need to fix names for example "book.png", and this name is false - you have png format of image/ and vektor siad that svg?